packages/@webex/plugin-meetings/src/media/properties.ts

```diff
@@ -212,6 +216,91 @@ export default class MediaProperties {
     };
   }
 
+  /**
+   * Checks if the given IP address is IPv6
+   * @param {string} ip address to check
+   * @returns {boolean} true if the address is IPv6, false otherwise
+   */
+  private isIPv6(ip: string): boolean {
+    return ip.includes(':');
```

Comment:
this one makes me laugh !

Reply:
as long as it's called with an ip address, then it does the job :-) If it ever needs to be more robust to handle non-ip strings like urls, then it can be expanded when needed.